src/watchman_watcher.js

```diff
@@ -271,6 +281,7 @@ WatchmanWatcher.prototype.close = function(callback) {
 
 function handleError(self, error) {
   if (error != null) {
+    console.log('[sane] Watchman: ' + error.message);
```

Comment:
I added this because I wasn't sure whether the common clients were plumbing this in, and wanted to err on the side of caution and make sure these are exposed in the short term.  I'm not going to campaign to keep it in for the long haul, as I agree that the library delegate this to the app.